src/rust/bitbox02-rust-c/src/util.rs

```diff
@@ -233,6 +248,26 @@ mod tests {
         rust_util_bytes(core::ptr::null(), 1).as_ref();
     }
 
+    #[test]
+    fn test_cstr_mut() {
+        let mut start = String::from("foo\0bar");
+        let mut cstr_mut = rust_util_cstr_mut(start.as_mut_ptr(), start.len());
+        assert_eq!(cstr_mut.len, start.len());
+        assert_eq!(cstr_mut.as_ref(), "foo");
+        let bytes_mut = unsafe { cstr_mut.as_bytes_mut() };
+        bytes_mut[0] = 'g' as u8;
```

Comment:
`b'g'` is probably more idiomatic than `'g' as u8`, but I guess both are fine.

Reply:
TIL